### ISSUE TITLE
Error Report: Cannot read property 'map' of null

### ISSUE THREAD

**Issue by hdzdwlrs** [urbit/bridge#554]
## Context

Logging into bridge with metamask

A wallet containing multiple planets

## Error

```
TypeError: Cannot read property 'map' of null
    at Object.Ok (https://bridge.urbit.org/static/js/main.b2800745.chunk.js:2263:2859)
    at InternalConstructor.matchWith (https://bridge.urbit.org/static/js/2.de79ebd6.chunk.js:33518:29)
    at https://bridge.urbit.org/static/js/main.b2800745.chunk.js:2263:2771
    at InternalConstructor.chain (https://bridge.urbit.org/static/js/2.de79ebd6.chunk.js:65133:14)
    at https://bridge.urbit.org/static/js/main.b2800745.chunk.js:2263:2750
    at Object.Mh [as useMemo] (https://bridge.urbit.org/static/js/2.de79ebd6.chunk.js:174847:7)
    at push.exports.useMemo (https://bridge.urbit.org/static/js/2.de79ebd6.chunk.js:170996:14)
    at Points (https://bridge.urbit.org/static/js/main.b2800745.chunk.js:2263:2728)
    at oh (https://bridge.urbit.org/static/js/2.de79ebd6.chunk.js:174592:7)
    at di (https://bridge.urbit.org/static/js/2.de79ebd6.chunk.js:175212:7)
```

In Bridge version 2.5.12.

**Comment by Fang-** [comment]
Alright, found out I can consistently reproduce this by first loading Bridge, and then (after pageload) changing the network that's selected in Metamask.  
This wasn't an issue before because changing the selected network in Metamask would make it reload the page. Metamask stopped doing recently.

Pushing in the simple fix for that today, but it's unclear that covers all cases of this error. #559 mentioned still encountering the issue after reloading the page.

In other words, this issue is a symptom of _some_ Ethereum connectivity issue. Why read calls would give `null` results instead of throwing exceptions, I do not yet understand...

**Comment by thelifeandtimes** [comment]
Closing out as stale. As of v3.2.14, Metamask has changed it's 'active network' model such that you can have any number of networks enabled at once and the network you are interacting with is selected at the time of transacting, rather than connecting.
